packages/php-wasm/node/src/lib/parse-worker-startup-options.ts

```diff
@@ -1,5 +1,3 @@
-import { WorkerStartupOptions } from '@php-wasm/abstract';
-
-export function parseWorkerStartupOptions(): WorkerStartupOptions {
+export function parseWorkerStartupOptions(): Record<string, string> {
```

Comment:
this is an ambitious type that doesn't match the code, because `JSON.parse` can produce strings and other values. e.g. if `WORKER_OPTIONS="\"test\""` then the result is a `string`

just noting in case you care. to resolve this the function could bail if the input isn't a JSON-encoded object.

```js
try {
	const data = JSON.parse(process.env['WORKER_OPTIONS']);

	return ('object' === typeof data && !Array.isArray(data) && null !== data)
		? data
		: {};
} catch ( e ) {
	return {};
}
```

Reply:
Good spot! Actually I realized this function isn't even used anywhere so let's just remove it